### PR TITLE
Improvements to the font selector and corrections for links highlighting

### PR DESCRIPTION
**Pull request by LucasTrg**

### CODE REVIEW

File: projects/lib/src/c4dt.css [hunk 1]
```diff
@@ -6,19 +6,33 @@
...
+.mat-body-1,
+.mat-select,
+.mat-select-value,
+.mat-tab-label,
+.mat-tab-link,
+.mat-form-field,
+.mat-stepper-vertical,
+.mat-form-field-appearance-legacy .mat-form-field-label,
+.mat-card,
+.mat-select-disabled {
```
Comment (tharvik) [inline projects/lib/src/c4dt.css]: It seems that there is an easier way to do it, did you try it?
https://stackoverflow.com/questions/43747518/angular-material-and-changing-fonts

**Comment by LucasTrg** [comment]
Indeed, it seems much cleaner, I'll try implementing the custom theming asap

**Comment by LucasTrg** [comment]
Wouldn't this kind of implementation rely on SASS ? if we were to switch c4dt.css to c4dt.scss, I'm not sure we could still import it in the other CSSs...

**Comment by ineiti** [comment]
If I understood correctly, you can do the `*`-trick in `.css`, but to use `@import` you'd need `scss`, which might effectively be overkill here.

**Pull request merged**